Blackjack/Deck.swift

```diff
@@ -10,14 +10,15 @@ import Foundation
 
 class Deck {
     var cards: [Card] = []
-    let titles = [Card.titles.jack, Card.titles.queen, Card.titles.king, Card.titles.ace]
-    
+    // Veli - I still dont think we should have titles as an enum in the Card class, if we do we need to include the numbers as well imo
+    let titles = ["2", "3", "4", "5", "6", "7", "8", "9", "10", Card.titles.jack.rawValue, Card.titles.queen.rawValue, Card.titles.king.rawValue, Card.titles.ace.rawValue]
```

Comment:
I like this format but maybe the number titles should be words to avoid confusion 
e.g. 
```suggestion
    let titles = ["two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", Card.titles.jack.rawValue, Card.titles.queen.rawValue, Card.titles.king.rawValue, Card.titles.ace.rawValue]
```

Reply:
:+1: